### DISCUSSION TITLE
Push to github issue

### DISCUSSION THREAD

**Discussion by diehendrick** [six7/figma-tokens#555] [General]
Hi, There. 

Thank you for the plugin, it's helpful.  unfortunately I got an issue when pushing to github,  all setting I follow based on your documentation, I used on another project it was no issue.

If the setting is incorrect, I will get error message with red background. but now I think when I try to push and commit this message will appear

<img width="856" alt="image" src="https://user-images.githubusercontent.com/1661591/160265642-e0d007e0-6971-4ff1-a756-479251c30729.png">

**Comment by six7** [comment]
Is the repository by chance an empty repository? There is a bug currently that requires the repository to have at least a Readme file, so it won't work with an empty one.
